README.ORG

```diff
@@ -21,7 +21,7 @@
 *** Install Hammerspoon
     You can use [[https://brew.sh/][brew]]:
     #+begin_src bash
-      brew --cask install hammerspoon
+    brew install hammerspoon
```

Comment:
That was my earlier sloppy addition. Apparently I got the order of arguments wrong, but I didn't want to create a whole 'nother pr just to fix that.